test/reorderError1.js

```diff
@@ -0,0 +1,11 @@
+/*global totalCount */
+
+QUnit.module( "Test call count" );
+// Then
+QUnit.test( "should be success", function( assert ) {
+	if ( window.sessionStorage ) {
+	    assert.equal( totalCount, 3 );
```

Comment:
how about checking `window.totalCount`? This way you won't need the jshint global definition.